spec/controllers/orphans_controller_spec.rb

```diff
@@ -49,6 +92,69 @@
       end
     end
 
+    context "when logged in as a suspended user" do
+      before { fake_login_known_user(suspended_user.reload) }
+      
+      it "shows the form for orphaning a work" do
```

Comment:
Should the tests in this `context` say something like "does not show the form for ..."? (similar question for the delete action below too)